packages/next-codemod/lib/handle-package.ts

```diff
@@ -41,7 +41,14 @@ export function uninstallPackage(
     command = 'remove'
   }
 
-  execa.sync(pkgManager, [command, packageToUninstall], { stdio: 'inherit' })
+  try {
+    execa.sync(pkgManager, [command, packageToUninstall], { stdio: 'inherit' })
+  } catch (error) {
+    throw new Error(
```

Comment:
Moved error handling to util function

Reply:
isn't the error will go into stdio?

Reply:
AFAIK yes as `stderr` but wouldn't swallow it, will be caught on `catch`.